task/hadolint/0.1/hadolint.yaml

```diff
@@ -33,14 +33,16 @@ spec:
       name: output-format
       type: string
   steps:
-    - image: 'ghcr.io/hadolint/hadolint:v2.8.0-alpine@sha256:6c4b7c23f96339489dd35f21a711996d7ce63047467a9a562287748a03ad5242'
+    - image: 'ghcr.io/hadolint/hadolint:v2.8.0-debian@sha256:50b0e60aa2b4aba5a26eeb4ad08c96ed7a828fca996632e29114aabea18345f4'
```

Comment:
@jandradap any reason for changing the image?

Reply:
I need bash to convert the string to array without problems with the linter.